lib/oppen/mixins.rb

```diff
@@ -6,22 +6,22 @@ module Mixins
     # Rotates circular array and triples its size.
     # This method is not for public use.
     #
-    # @param arr [Array]
-    # @param offset [Integer] Rotation amount
+    # @param arr    [Array]   The circular array.
+    # @param offset [Integer] Rotation amount.
     #
-    # @return [Array<Array, Integer, Integer>] upsized array, lhs, rhs
+    # @return [Array<Array, Integer, Integer>] upsized array, lhs, rhs.
```

Comment:
We're also inconsistent when it comes to parametric type definitions in our docs. We sometimes use `< >` and in other times we `[ ]`. [Pick one](https://m.media-amazon.com/images/I/61D-xUFCtXL._AC_UF1000,1000_QL80_.jpg)